src/main/java/com/kodilla/ecommercee/EcommerceeApplication.java

```diff
@@ -1,12 +1,14 @@
 package com.kodilla.ecommercee;
 
+import lombok.Getter;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
 public class EcommerceeApplication {
 
-    public static void main(String[] args) {
+    public static void main(String[] args)
```

Comment:
po co ta zmiana?

Reply:
Jeśli chodzi o import lombok.Getter to nie dopatrzyłem usunięcia, sprawdzałem czy działa.